doc/source/install.rst

```diff
@@ -68,30 +66,15 @@ installed using Conda by running::
     $ conda install --file requirements.txt
 
 After navigating to inside the `tomopy` directory, you can install TomoPy by
-activating the Conda base environment and running the install script::
+building/compiling the shared libraries and running the install script::
 
-    $ source activate
-    (base) $ python setup.py install
-
-Activating the conda environment ensures that `setup.py` will know where to
-look for the mkl headers. If you didn't use Conda to install the mkl library,
-you may need to read the section below.
+    $ python build.py
+    $ pip install .
```

Comment:
Uninstalling something installed with `pyhon setup.py install` is difficult because you have to manually delete the files. `pip` makes it easy because it keeps metadata which lets it remove packages automatically.